### ISSUE TITLE
dotnet20 env build error

### ISSUE THREAD

**Issue by soamvasani** [fission/fission#359]
Using the `microsoft/dotnet:2.0.0-sdk` docker image to build the dotnet20 environment server, I got this error:

```
# dotnet restore
MSBUILD : error MSB1011: Specify which project or solution file to use because this folder contains more than one project or solution file.
```

@joalmeid

**Comment by joalmeid** [comment]
In the dotnet20 env folder we opted by leaving both the fission-dotnet20.csproj and the dotnet20.sln. Dotnet restore, requires you to explicit name the project (if you have multiple proj/sln files). So you basically have to use "dotnet restore fission-dotnet20.csproj".

Having the .sln, will allow people to also open the code in VIsual Studio, and take a look at the dotnet20 env.

I can change the documentation (recommending doing a dotnet restore fission-dotnet20.csproj).

**Comment by soamvasani** [comment]
Fixed by #365, thanks.
